workspaces/blackduck/plugins/blackduck-backend/config.d.ts

```diff
@@ -0,0 +1,12 @@
+export interface Config {
+  blackduck: {
+    /**
+     * @visibility frontend
+     */
+    host: string;
+    /**
+     * @visibility secret
+     */
+    token: string;
+  };
+}
```

Comment:
Based on the config mentioned in the [README](https://github.com/backstage/community-plugins/blob/main/workspaces/blackduck/plugins/blackduck-backend/README.md#global-config), I think we'd want to make a few changes.
```suggestion
export interface Config {
    blackduck: {
      /**
       * The default host name.
       * @visibility frontend
       */
      default: string;
      /**
       * An array of Black Duck hosts.
       */
      hosts: Array<{
        /**
         * Host name identifier.
         * @visibility frontend
         */
        name: string;
        /**
         * The host URL for the Black Duck instance.
         * @visibility frontend
         */
        host: string;
        /**
         * API token for authentication.
         * @visibility secret
         */
        token: string;
      }>;
    };
  }
```

Reply:
@04kash 
Thanks